src/applications/hca/routes.jsx

```diff
@@ -1,31 +1,15 @@
 import { createRoutesWithSaveInProgress } from 'platform/forms/save-in-progress/helpers';
-import environment from 'platform/utilities/environment';
 
 import formConfig from './config/form';
 import HealthCareApp from './HealthCareApp.jsx';
-import IDPage from './containers/IDPage';
-
-const formRoutes = createRoutesWithSaveInProgress(formConfig);
-
-// If the route's path ends in `introduction` then the form system will treat it
-// as an Introduction page (ie it won't add the Title component or navigation
-// elements) which is what we want in this case.
-const idFormRoute = {
-  path: 'id-introduction',
-  component: IDPage,
-  key: 'id-introduction',
-};
 
 const routes = {
   path: '/',
   component: HealthCareApp,
   indexRoute: {
     onEnter: (nextState, replace) => replace('/introduction'),
   },
-  // do not allow the idFormRoute in production
-  childRoutes: environment.isProduction()
-    ? formRoutes
-    : [idFormRoute, ...formRoutes],
```

Comment:
It looked like your original intent was to add the additional routes in the routes module. My take was that the additional pages should be obvious from the form config, so they're handled in there instead. Do you think that approach makes sense?

Reply:
I didn’t even know it was possible to do it the way you did it. We’ll need to keep the route path ending in “introduction” however, otherwise the form system will add additional boilerplate header/footer markup to the IDPage component and we don’t want that. It’s hacky, but paths that end in “introduction” are treated differently. I’d proposed a change to the form system to make it more explicit but got shot down :)